frontend/packages/kubevirt-plugin/integration-tests/tests/vm.consoles.scenario.ts

```diff
@@ -0,0 +1,57 @@
+import { browser, ExpectedConditions as until } from 'protractor';
+import { testName } from '@console/internal-integration-tests/protractor.conf';
+import { 
+  click, 
+  selectDropdownOption,
+  withResource, 
+} from '@console/shared/src/test-utils/utils';
+import {
+  consoleTypeSelector,
+  consoleTypeSelectorId,
+  serialDisconnectButton,
+  serialReconnectButton,
+  vncSendKeyButton,
+} from '../views/consolesView';
+import { PAGE_LOAD_TIMEOUT_SECS } from './utils/consts';
+import { VirtualMachine } from './models/virtualMachine';
+import { vmConfig, getProvisionConfigs } from './vm.wizard.configs';
+import { ProvisionConfigName } from './utils/constants/wizard';
+
+describe('KubeVirt VM VNC/Serial consoles', () => {
+  const leakedResources = new Set<string>();
+
+  const provisionConfigs = getProvisionConfigs();
+  const configName = ProvisionConfigName.CONTAINER;
+  const provisionConfig = provisionConfigs.get(configName);
+
+  provisionConfig.networkResources = [];
+  provisionConfig.storageResources = [];
+
+  const testVMConfig = vmConfig(
+    configName.toLowerCase(),
+    testName,
+    provisionConfig,
+  );
+  const vm = new VirtualMachine(testVMConfig);
+
+  it('VM VNC/Serial Console is connected', async () => {
+    await withResource(leakedResources, vm.asResource(), async () => {
+      await vm.create(testVMConfig);
+
+      // verify Serial Console is connected
+      await vm.navigateToConsoles();
+      await browser.wait(until.presenceOf(consoleTypeSelector));
+      await click(consoleTypeSelector);
```

Comment:
why are we clicking twice here?

Reply:
It has to click twice based on my test result, otherwise it does not work(timeout).

Reply:
hm that's weird. Can you put a small note there? possibly with a reason why it is happening

Reply:
Added a comment, don't know the exact reason, but I do remeber we have the same issue before, it seems to be the cromedriver issue.